### ISSUE TITLE
SpecialArgumentsCollection has outdated argument names

### ISSUE THREAD

**Issue by jamesemery** [broadinstitute/barclay#120]
The arguments "showHidden" and "arguments_file" in `SpecialArgumentsCollection` in Barclay have the old argument style and should be changed to the new argument style.

**Comment by magicDGS** [comment]
This is a disruptive change for downstream users not using GATK, and I would like to maintain some downstream tools based on barclay without changes in the command line (so disruptive for my users).

I suggest that, if this happen, it will be for barclay 2.0 - otherwise, I would like to make configurable the special arguments by the API user. I hope that @cmnbroad agrees on that...

**Comment by cmnbroad** [comment]
Yes, ideally these names would be configurable by the caller, but the parser depends on knowing what they are; making them configurable means we'd need some way to pass them in. So for now well probably just have to update them as part of 2.0.
